core/block.js

```diff
@@ -1415,15 +1388,14 @@ Blockly.Block.newFieldImageFromJson_ = function(options) {
   var height =
     Number(Blockly.utils.replaceMessageReferences(options['height']));
   var alt = Blockly.utils.replaceMessageReferences(options['alt']);
-  var flip_rtl = Blockly.utils.replaceMessageReferences(options['flip_rtl']);
-  return new Blockly.FieldImage(src, width, height, alt, flip_rtl);
+  return new Blockly.FieldImage(src, width, height, alt);
```

Comment:
Because you're still using it: https://github.com/LLK/scratch-blocks/blob/develop/blocks_horizontal/event.js#L49